tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -937,4 +937,16 @@
     <value>X64 process running on X64 machine</value>
     <comment>Message text when the process is x64 CPU architecture running on an X64 CPU machine.</comment>
   </data>
+  <data name="AliasDisabledDialogButtonText" xml:space="preserve">
+    <value>Go to Advanced app settings</value>
+    <comment>Content Dialog button text which on click opens the Advanced App settings page in Windows Settings app.</comment>
+  </data>
+  <data name="AliasDisabledDialogContent" xml:space="preserve">
+    <value>Go to Advanced app settings &gt; App execution alias &gt; Project Ironsides &gt; Enable the toggle and try to launch PI as admin again.</value>

```

Comment:
I wonder if this is clear enough, given that PI's own Settings also has an Advanced settings section? Consider changing this to "Windows Settings &gt; Apps &gt; Advanced app settings"